src/utils/formatNumber.ts

```diff
@@ -74,10 +74,10 @@ export const formatCurrency = (number: string | number, currency: string, maxLen
     result = getCurrencyFormatter(currency, true, 2).format(float)
   }
 
-  return result.replace(/^(\D+)/, '$1 ')
+  return result.replace(/^(\D+)/, '$1 ')
```

Comment:
This is what it looks like for me now 🤔

<img width="357" alt="Screenshot 2024-08-05 at 12 47 27" src="https://github.com/user-attachments/assets/50c658b0-8cc4-444f-9b97-515e5e261248">

Reply:
I made the space thinner (it was a thin space, now a hair-width space).

Reply:
Interesting, TIL!

Reply:
This also only seems to apply if the currency symbol is in front of the value.